Package.swift

```diff
@@ -36,8 +36,8 @@ let package = Package(
         .package(name: "MobilePassiveData",
                  url: "https://github.com/NU-MSS/MobilePassiveData-SDK.git",
                  from: "1.5.5"),
-        .package(url: "https://github.com/NU-MSS/AssessmentModelKMM.git",
-                 from: "1.0.0"),
+		.package(name: "AssessmentModel", url: "https://github.com/NU-MSS/AssessmentModelKMM.git",
```

Comment:
We should use a tag here since AssessmentModeilKMM isn't under active development by us